app/views/thredded/topics/active.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :thredded_page_title, t('thredded.nav.active_topics') %>
+<% content_for :thredded_page_id, 'thredded--topics-index' %>
+<% content_for :thredded_breadcrumbs, render('thredded/shared/breadcrumbs') %>
+
+<%= thredded_page do %>
+  <div class="thredded--svg-definitions ">
```

Comment:
Looks like these are duplicated. Perhaps extract to a shared `thredded/topics/_list_svg_definitions.html.erb` partial?